client/src/utils/fight/setupFight.ts

```diff
@@ -464,6 +475,19 @@ const setupFight: (
       throw new Error('Step not found');
     }
 
+    // Display step's brute in HUD
+    if ('b' in step && step.a !== StepType.AttemptHit) {
+      void setHUDFocus(app, renderer, fighters, step.b, speed, isClanWar);
+    }
+    // Display step's fighter in HUD
+    if ('f' in step) {
+      void setHUDFocus(app, renderer, fighters, step.f, speed, isClanWar);
+    }
+    // Display step's target in HUD
+    if ('t' in step && step.a !== StepType.Bomb) {
+      void setHUDFocus(app, renderer, fighters, step.t, speed, isClanWar);
+    }
```

Comment:
Using `Object.hasOwn` is safer, as it has less edge cases

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn

Reply:
I had to keep the "in" to get eslinted though